core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/controller/Workflow.scala

```diff
@@ -24,6 +24,7 @@ import scala.collection.mutable.ArrayBuffer
 
 class Workflow(
     workflowId: WorkflowIdentity,
+    pythonOperators: mutable.Set[OpExecConfig],
```

Comment:
I think this is a redundant information. `operatorToOpExecConfig` should already contain all OpExecConfig already. And whether it is python or not should be determined from `operatorToOpExecConfig`. I suggest we do not pass `pythonOperators` again.

Reply:
By only looking at `OpExecConfig`, we cannot say whether an operator is a python operator or not. Previously we check the `IOperatorExecutor` but now we can't.

Reply:
then isn't it better to add some information to `OpExecConfig`?